### DISCUSSION TITLE
Who wants to be hired? (Dec 2021)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#32094] [Ideas]
# Latest post (use this)
- https://github.com/vercel/next.js/discussions/35839

# What is this?
This discussion is for Next.js developers looking for work.

# Candidates

Share your information if you are looking for work. Please use this format:

  Location:
  Remote:
  Willing to relocate:
  Technologies:
  Résumé/CV:
  Website:
  Email:

Good luck with interviewing!

---

*If you're looking for last month's discussion, see #31541. At least one developer has gotten hired from posting their profile!*

If you're a company looking to post a role, use #32093.

**Comment by rbourdon** [comment]
Location: **Salem, Massachusetts USA**
Remote: **Yes**
Willing to relocate: **No**

Technologies: **Next.js - React - Framer Motion - GraphQL - PostgreSQL - Sendgrid - GraphCMS - Illustrator - Blender**

Résumé/CV: **https://www.rorybourdon.com/Rory_Bourdon_Resume_2021.pdf**
Website: **https://www.rorybourdon.com/**
Email: **rorybourdon@gmail.com**

**Reply by Itshalffull** [reply]
Hey Rory,

Love your website! We're currently looking for a part time Next.js developer familiar with GraphQL and would love to chat!  We're a small non-profit in rural Vermont looking to build new governance/decision making systems.

Warmth,
Matt

matt (at) cml (dot) me

**Reply by rbourdon** [reply]
Hi Matt,

Thank you for the kind words :heart: I sent you an email.

**Comment by holly-codes** [comment]
Location: **Connecticut, USA**
Remote: **Yes**
Willing to Relocate: **No**

Technologies: **NextJS, ReactJS, SASS, HTML, JavaScript, GraphQL**

Resume/CV: **https://holly-codes.com/Holly-Burinda-Resume-2021.pdf**
Website: **https://holly-codes.com**
Email: **hello@holly-codes.com**

**Reply by duosana** [reply]
Hey @holly-codes 

Your profile looks amazing and I have referred you to our recruiting team here at Sana Benefits. We are a health insurance company that helps small businesses and medium sized companies. We have a wonderful community of developers and a culture of growth and transparency where we enjoy supporting/mentoring each other and promoting from within. We offer competitive salaries and amazing benefits - we are a benefits company after all!

Here is our job posting if you would like more information: https://jobs.lever.co/sanabenefits/943da474-5feb-4e67-bb3d-2a9be0f6b127?lever-via=1BLCQyk8VY

A member of our recruiting team will be reaching out shortly. We look forward to connecting if you are interested!

**Comment by leonidlewis** [comment]
I am Leonid from HK. I am sure to be your team member in your team.
- Location : Hong Kong-HK
- Remote: Yes
- Willing to relocate: No
- Technologies : Nextjs - Redux-saga - Material UI - Ant Design - Tailwind - Design
- https://dev-about.netlify.app/
- Email: leonid.lewis@gmail.com

**Reply by droegier** [reply]
Hi Leonid, please have a look at our opening.
https://jurata.join.com/jobs/3443739-react-developer

**Comment by abayomi1196** [comment]
Location: Abuja, Nigeria
Remote: Yes
Willing to relocate: Yes
Technologies: React, NextJS, Typescript, Vue, NuxtJS, GraphQL
Résumé/CV: [My Resume / CV](https://drive.google.com/file/d/18xBgPxLolsORe-qmI6xEurR0WBRVErQf/view?usp=drivesdk) 
Website: [My Portfolio](https://portfolio-ayodele.vercel.app/) 
Email: samabayomi301@gmail.com

**Comment by VladRDV** [comment]
Location: Belgrade - Serbia
Remote: Yes
Willing to relocate: No
Technologies: React, Next.js, React Native, Express.js, TypeScript, Three.js, Firebase, Vercel, Stripe
Résumé/CV:  [my CV](https://drive.google.com/file/d/1lKU38Bev1DSQeBWd3ybX0wL0O8kRNuSm/view)
Website: https://vradov.space/
Email: vlad.radovanovich@gmail.com

**Reply by droegier** [reply]
Hi Vladimir, please have a look at our opening. It says Porto, but we are open to work remote as well.

https://jurata.join.com/jobs/3443739-react-developer

**Reply by alopezmontano** [reply]
Hi Vlad!
We are a Startup building a HR tech platform, I have seen your profile and I think our opening could be interesting for you...
Please take a look at: https://www.linkedin.com/jobs/view/2846908950/?alternateChannel=search&refId=ceV5Q9nQ%2FlwAH%2BRWxjZ7Mw%3D%3D&trackingId=1bUxN3wqxc9ytZdpl%2BVJzw%3D%3D)
If you want to know more just tell and we'll have a talk.
BR,
Antonio

**Comment by ninsau** [comment]
Location: Accra, Ghana
Remote: Yes
Willing to relocate: Yes
Technologies: NextJs, React, TypeScript, JavaScript, AWS Amplify, FireBase, MaterialUI
Résumé/CV: https://drive.google.com/file/d/1qNLnlu_48uqQ8maZdRfAzCChXOvDel97/view?usp=sharing
Email: cassidyblay@gmail.com

**Comment by mikealche** [comment]
Location: Buenos Aires, Argentina
Remote: Yes
Willing to relocate: No
Technologies: NextJs, React, TypeScript, JavaScript,  Node.js, Prisma, GraphQL, tRPC, PostgreSQL, MongoDB
Résumé/CV: https://www.mikealche.com/
Email: mikealche@gmail.com

[These](https://www.mikealche.com/software-development/how-to-implement-search-by-color-when-all-you-have-is-a-good-coffee) are [some blog posts](https://www.mikealche.com/software-development/a-humble-guide-to-database-schema-design) that you [may like](https://www.mikealche.com/software-development/learn-react-animations-by-creating-a-stripe-inspired-menu)

**Comment by straxico** [comment]
Hello world ☺️

Location: iran
Remote: Yes
Willing to relocate: No
only part time
only can transfer money with cryptocarency

i have full job in iran.The value of money in Iran is very small. Iran has a special economy and my annual income is about $ 2000. That's why working with other countries is so economical for me, and I can work part-time for you for a small amount of money.

Technologies: React, Next.js,TypeScript and ...

you can find me everywhere with search "straxico"
Linkedin: https://www.linkedin.com/in/straxico
Email: straxico@gmail.com

**Comment by crackwebai1024** [comment]
Hi. everyone.
Remote: Yes
Location: China
Willing to relocate: No
working timezone: client's timezone(including UK, EST, CST, PST)
Looking for: fully remote job
mainskills: 
 - Frontend 
     (SPA - React, Angular, Vue, Svelte)
     (SSR - Next, Nuxt, Sapper)
     (UI - material ui, tailwind CSS, Chakra UI, styled-components)
 - Backend (Node - express, Python - django, aws Lambda(chalice, nodejs))
website: http://mingxing.vercel.app	
email: piaobristar@gmail.com

**Comment by iamsharzil** [comment]
👋  Hi, I’m Sharzil Shaikh 

- Location: India
- Remote: Yes
- Willing to relocate: No 
- Technologies: NextJs, React, TypeScript, GraphQL, JavaScript, React Testing Library, Redux, MaterialUI
- CMS: Magento, WordPress, Shopify
- Résumé/CV: [Resume](https://drive.google.com/file/d/1_ZRXX6BXatvXhw19O9W0qTSZbpmGDm_Y/view)
- LinkedIn: https://www.linkedin.com/in/iamsharzil/
- Email: iamsharzil@gmail.com
- Blogs:  https://medium.com/@iamsharzil
- Twitter: https://twitter.com/SharzilShaikh
- Projects - 
    1. https://cms-magento.vercel.app/ 
    2. https://www.zenius.net/ 
    3. https://www.discoverpilgrim.com/
    4. https://www.californiadesignden.com/

**Comment by TopDev130** [comment]
Hello everyone,
This is an experienced senior full stack developer.

Location: Hong Kong
Remote: yes
Willing to relocate: not sure
Technologies: Next, React, Typescript, Redux, Redux saga & thunk, Bootstrap, Material UI, Ant Design, Tailwind
Résumé/CV: [Resume](https://www.dropbox.com/sh/viz7dmbxkvyndl2/AABwutvPTllslGH9XTAH9nRTa?dl=0)
Email: hideo.prodev@gmail.com

Thank you.

**Comment by yarnJ** [comment]
Location: Canada, Ontario
Remote: yes
Willing to relocate: no
Technologies: React js, Redux-Saga/Thunk, Javascript, Gatsby js, Rest, Typescript, MongoDB, Firebase
Résumé/CV:  https://www.linkedin.com/in/james-yarn-308780227/
Website: https://yarnj.github.io/jamesyarn/
Email: yarnj33@gmail.com

**Comment by albertdugba** [comment]
Location : Ghana, Accra
Remote:Yes
Technologies used: Nextjs, react,  CSS-In-Js, typescript and graphql 
Portfolio: https://albertdugba.dev
Email: albert.dugba@gmail.com

**Comment by prmlee** [comment]
Location: Hong Kong
Remote: Yes
Willing to Relocate: No
Technologies:
🔸 HTML, CSS, Bootstrap, React, Vue.js, Next, Nuxt, React Native, JQuery
🔸 Node.js, ExpressJS, Typescript, Python, Django, Flask, Bottle
🔸 MySQL, PostgresSQL, MongoDB, TypeORM, Sequelize
🔸 GCP, AWS, Google Firebase
🔸 Github, Bitbucket
🔸 Jira, Trello
🔸 Jenkins
🔸 RESTful, GraphQL (Apollo)
🔸 Mocha/Chai
🔸 Blockchain, NFT-marketplace, web3.js
🔸 three.js
Resume: https://docs.google.com/document/d/e/2PACX-1vTmFRSiVQM3EbHou_ztnoZDgOMUB_CqWmxn9UsnbIDqNqwD4nfM0Iv2XlTOrfykztBpks0L9RNlfKMF/pub
Email: promotionlee.bs@gmail.com

**Comment by yarnJ** [comment]
Location: Toronto on Ontario on Canada
Remote: sure
Willing to relocate: no
Technologies: javascript frameworks including react/redux/thunk, saga, Next, Nuxt etc and familiar with backend.
Résumé/CV: https://www.linkedin.com/in/james-yarn-308780227/
Website: https://yarnj.github.io/jamesyarn
Email: yarnj33@gmail.com

**Comment by crackwebai1024** [comment]
[FOR HIRE][REMOTE][FULL-STACK prefer FRONTEND]

Remote : Yes
Willing to relocate: No
working timezone : client's timezone (including UK, EST, CST, PST)
Looking for : fully remote job (full-time)
mainskills : 
 - Frontend 
     (SPA - React, Angular, Vue, Svelte)
     (SSR - Next, Nuxt, Sapper)
     (UI - material ui, tailwind CSS, Chakra UI, styled-components, vuetify)
 - Backend (Node - express, Python - django, aws Lambda(chalice, nodejs))
portfolio : http://mingxing.vercel.app

**Comment by developerisgood** [comment]
Hello 
Remote: yes
Willing to relocate: No
working timezone: flexible time zone
Looking for: full-time
Resume: https://drive.google.com/file/d/1sdUb4xAeHCxnAzs4N0gLqqWA-InjoA7u/view?usp=sharing  
Email: micael.dev216@gmail.com
Website: https://michaeldev.herokuapp.com/
I am a full-stack and blockchain developer.
👍 8 + years experience as Full stack Developers and during this tenure, I have worked on the different frameworks and CMS.
💪 STRONG POINT:
★★★★★ Blockchain:  (Smart Contract/Solidity/Ethereum/Bitcoin/Algorand)
★★★★★ Front-end:  React/Redux/Thunk/NextJS + Material UI, Angular, Vue, Woo-commerce
★★★★★ Backend: NodeJS, Express, Python, Django, PHP, Laravel, Strapi.
★★★★★ Database: MySQL, Postgres, MongoDB
★★★★★ NFT game development
I hope we can discuss more details on chat.
Please send me a message and we can discuss further detail.
Thanks in advance.
I am available to work in any time zone and can start the work immediately.

**Comment by Samuelsonvs** [comment]
Hello.

Location: Turkey
Remote: Yes
Willing to relocate: No
Technologies: Next, React, TypeScript, Node, Tailwind CSS, Supabase, GraphQl
Website: https://msatali.com
Email: m.sametatali@gmail.com

Thank you.

**Reply by alopezmontano** [reply]
Hi!
We are a Startup building a HR tech platform, I have seen your profile and I think our opening could be interesting for you...
Please take a look at: https://www.linkedin.com/jobs/view/2846908950/?alternateChannel=search&refId=ceV5Q9nQ%2FlwAH%2BRWxjZ7Mw%3D%3D&trackingId=1bUxN3wqxc9ytZdpl%2BVJzw%3D%3D)
If you want to know more just tell and we'll have a talk.
BR,
Antonio

**Comment by azakero** [comment]
Hello. 

Location: Bangladesh
Remote: Yes
Willing to relocate: Yes
Technologies: 
Backend:
- NodeJS
- ExpressJS || Koa
- MongoDB || Mongoose || PostgreSQL || Firebase
- Jest
- JWT || Git || Postman
- Linux || Ubuntu

Frontend:
- CSS3 || SASS
- Vanilla Javascript || jQuery
- ReactJS || Redux || NextJS
- Material UI || Tailwind CSS
- FabricJS || Apex Charts

Resume/CV: https://drive.google.com/file/d/1oKX6L4-or6hAbVxP01qf68uio9-wRnzD/view
Website: https://www.zaker.dev
Email: gloryfaircompany@gmail.com

Thank you.

**Comment by rehanpunjwani** [comment]
Remote: Yes
Willing to relocate: Yes
Technologies: 
React, Redux, Javascript, Node, Python, Flask, Next.js, Nestjs, Express.js, TypeScript, Stripe, Mongo, Postgres, Prisma, Unity3D, MySQL, Linux, Git, Postman, Docker, Kubernetes, GCP, 
Résumé/CV: https://shorturl.at/jlrEQ
Website: https://rehanpunjwani.github.io/
Email: rehanpunjwani@gmail.com

**Comment by mtergel** [comment]
Location: **Ulaanbaatar, Mongolia**
Remote: **Yes**
Willing to relocate: **Maybe**

Technologies: Next.js - GraphQL - Tailwind CSS - React Spring - Framer Motion - Firebase - Supabase - AWS Amplify

Résumé/CV: https://drive.google.com/file/d/1SYRqtGmf7uwiKEwAXplrYFXapX4bu5VG/view
Website: https://portfolio-tergelm.vercel.app
Email: tergelm@gmail.com

**Comment by Christopher-Stevers** [comment]
Location: Kitchener, Ontario
Remote: Yes
Willing to relocate: No.
Technologies: HTML, CSS, JavaScript, React.js, Next.js, Redux, Sass, Node.js, Express, MongoDB, Webpack, PostgreSQL, Git.
Website: [https://devstevers.com](https://devstevers.com)
Resume: [https://docs.google.com/document/d/1Oq5E2CZu8XSuWbmL8iF1CWhD5UHIQLdCPOiEiQvxu5Q/edit?usp=sharing](https://docs.google.com/document/d/1Oq5E2CZu8XSuWbmL8iF1CWhD5UHIQLdCPOiEiQvxu5Q/edit?usp=sharing)
Email: christopherstevers@protonmail.com

**Comment by codejesse** [comment]
Location: Port Harcourt, Nigeria 
Remote: Yes
Willing to relocate: No 
Technologies: HTML, CSS, JavaScript, Reactjs, Nextjs, Firebase, Git (Front-End)
Résumé/CV:  [Jesse's Resume (2).pdf](https://github.com/vercel/next.js/files/7836223/Jesse.s.Resume.2.pdf)
Website: https://codejesse.vercel.app/
Email: jessebeke10@gmail.com

**Comment by kolpaja** [comment]
Location: Albania/Europe
Remote: yes
Willing to relocate:yes
Technologies: MERN stack
Résumé/CV: [Sokol-Paja-CV .pdf](https://github.com/vercel/next.js/files/9377746/Sokol-Paja-CV.pdf)
Website: https://kolpaja.github.io/
Email: kol.paja@gmail.com

**Comment by Brettkgamble** [comment]
Location: Edmonton/Canada
Remote: Yes
Willing to relocate: No
Technologies: HTML, CSS, JavaScript, React.js, Next.js, Sanity, Redux, PostgreSQL, Git, BitBucket
Resume:  
[Brett Gamble.pdf](https://github.com/vercel/next.js/files/12654041/Brett.Gamble.pdf)
Website: [www.brettkgamble.com](url)
Email: [Brettkgamble@gmail.com](url)

**Comment by leonidlewis** [comment]
What's this?
